packages/generators/generators/lib/plops/utils/get-formatted-date.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+module.exports = (date = new Date()) => {
+  return new Date(date.getTime() - date.getTimezoneOffset() * 60000)
+    .toISOString()
+    .split('.')[0]
+    .replace(/[-T:]/g, '_');
```

Comment:
```suggestion
  return new Date(date.getTime() - date.getTimezoneOffset() * 60000).toJSON().replace(/[-:]/g, ".").replace(/\....Z/, "");
```
Example: `2022.05.10T00.00.00.name-of-my-migration.js`
I believe this is what we currently recommend in [our documentation](https://docs.strapi.io/developer-docs/latest/developer-resources/database-migrations.html), but I can't say I like it.

However, the most commonly used by far, which I would personally prefer if we're going to make a change, is without any separators like:

```suggestion
  return new Date(date.getTime() - date.getTimezoneOffset() * 60000).toJSON().replace(/[-:T]/g, "").replace(/\....Z/, "");
```
Example: `20220510123456_name-of-my-migration.js`

@Convly @Bassel17 thoughts?

Reply:
I would say, let's stick with what we have currently documented, and if we want to change the format we'll update the doc & the generator at the same time.